### ISSUE TITLE
RPD hotkey disabling does not work when using the MoMMI RPD.

### ISSUE THREAD

**Issue by Eneocho** [d3athrow/vgstation13#31637]
#### Description of issue
When you shift and scroll the wheel you can change the pipe layer. Normally, shift clicking examines the item, but shift clicking while using the RPD just places a pipe, this allows for fast pipe layering since you can keep the shift on and just scroll and click.
This feature does not happen with MoMMI RPDs for some reason. I had to pipe very slowly since it destroyed my keyboard/mouse flow.
#### Difference between expected and actual behavior
Expected the MoMMI RPD Module to also have the hotkey overriding that normal RPDs do.
Got very slow piping since I'm not used to the slow way anymore.
#### Steps to reproduce
Use an RPD as a human.
Select pipe.
Press shift and click. You'll place a pipe instead of examining.
Still shifting, scroll and click. You'll place a pipe on the previous/next layer.

Use the RPD module as MoMMI.
Select Pipe.
Press shift and click. You'll examine instead of placing a pipe.
#### Length of time in which bug has been known to occur
Probably a long time.

[bugfix] [consistency] possibly an [oversight]

**Comment by pedr69** [comment]
hol up
you're saying i've been clicking the 4 pixel wide pipes to change layers for no reason all this time?
sheeeit

**Comment by Eneocho** [comment]
Yes. Fastlayering saves you a fuckton of time, at least 15 seconds per layer group.
